pkg/conditions2/getter.go

```diff
@@ -20,7 +20,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
-	vmopv1 "github.com/vmware-tanzu/vm-operator/api/v1alpha2"
+	vmop "github.com/vmware-tanzu/vm-operator/api/v1alpha2"
```

Comment:
I prefer to keep the existing alias. Prior commits tried to standardize on `vmopv1` where we could so the v1a1 and v1a2 code has few differences.

Reply:
I kept stumbling across the vmopv1 alias and double checking the import to verify if it is alpha1 or alpha2. 
Was hoping to eventually get the aliases sorted out across the codebase without an all encompassing PR to replace all occurrences at once.